api/openapi/api.yaml

```diff
@@ -179,3 +179,146 @@ paths:
           description: Check schematic log
       tags:
         - Model Operations
+  /storage/projects:
+    get:
+      summary: Get all storage projects the current user has access to
+      description: Gets all storage projects the current user has access to, within the scope of the 'storageFileview' attribute.
+      operationId: api.routes.get_storage_projects
+      parameters:
+        - in: query
+          name: input_token
+          schema:
+            type: string
+            nullable: false
+          description: Token
+          example: Token
+          required: true
+        - in: query
+          name: syn_master_file_view
+          schema:
+            type: string
+            nullable: false
+          description: synapse ID of master file view. 
+          example: syn23643253
+          required: true
+        - in: query
+          name: syn_manifest_file_name
```

Comment:
Change syn_manifest_file_name to manifest_file_name

Reply:
Okay. Got it! I have deleted this parameter for this endpoint.